src/dotnet/APIView/APIViewWeb/LeanModels/ChangeHistory.cs

```diff
@@ -46,7 +46,8 @@ public enum CommentChangeAction
     }
 
     public abstract class ChangeHistoryModel
-    {
+    {   
+        public string ChangeAction { get; set; }
```

Comment:
Why do we need this additional property? Can't we use APIRevisionChangeAction enum to compare the changeAction?

Reply:
Yes we definitely should not be adding this.